edx_django_utils/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '1.0.3'
+__version__ = '1.0.4'
```

Comment:
I just use the last for bug fixes.  For me, this would be `1.1.0`.

Reply:
Updated to 1.1.0